tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,36}, flake8
+envlist = py{27,36}, flake8, docs
```

Comment:
Good question, possibly tox ignores a missing environment in the list if there's a section for it.